workspace/apps/pidp/src/app/features/portal/portal.page.ts

```diff
@@ -80,6 +93,38 @@ export class PortalPage implements OnInit, AfterViewInit {
     this.alerts$ = this.resource
       .getProfileStatus(this.partyService.partyId)
       .pipe(map((profileStatus) => profileStatus?.alerts ?? []));
+    if (this.previousUrl.split('/').includes(Constants.newUserURL)) {
+      this.showNewUserPopup();
+    }
+  }
+
+  private showNewUserPopup(): void {
+    const data: DialogOptions = {
+      title: 'Account information completed',
```

Comment:
As this DialogOptions title is used for different purposes in different component can we define in another file as constants like
dialog-options-constants.ts
portal_page_dialog : {
title : "XXX",
content : "YYY"
},
endorsement_complete_dialog: {
title : "pqr,
content: "tttt"
}
......